src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -6,72 +6,74 @@
 from azure.cli.core.commands import cli_command
 from azure.cli.core.commands.arm import cli_generic_update_command
 from azure.cli.core.util import empty_on_404
+from azure.cli.testsdk import get_active_api_profile
 
 from ._constants import WEBHOOK_API_VERSION
 from ._format import output_format
 from ._factory import get_acr_service_client
 
-cli_command(__name__, 'acr credential show',
-            'azure.cli.command_modules.acr.credential#acr_credential_show',
-            table_transformer=output_format, exception_handler=empty_on_404)
-cli_command(__name__, 'acr credential renew',
-            'azure.cli.command_modules.acr.credential#acr_credential_renew',
-            table_transformer=output_format)
+if get_active_api_profile() == 'latest':
```

Comment:
If anything, I would try:
`if supported_api_version(PROFILE_TYPE, min_api='2017-03-10-profile'):`

This ensures that it won't be supported by your profile (2017-03-09-profile). However, @derekbekoe should really weigh in on this.

But comparing against latest is just asking for bugs. As soon as a new profile is stamped, even if it supports those services, those commands will vanish because they aren't "latest".